Automattic/woocommerce-payments
# Add JCB as a coming soon to the list of available payment methods

#### Changes proposed in this Pull Request
In order to promote the JCB payment method to Japanese merchants with one of the upcoming releases we added it to the list of available payment methods with a coming soon state.

<img width="1148" alt="Screenshot 2023-08-04 at 1 00 41" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/434f66d0-deab-48de-975b-aa4a668356dd">

<img width="1077" alt="Screenshot 2023-08-04 at 1 00 48" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/421ead77-555d-4f00-9628-fe4a75b0d26b">

<img width="376" alt="Screenshot 2023-08-04 at 1 00 30" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/f0775513-ec18-4716-9bab-2753eb26b4a2">

#### Testing instructions
- Navigate to `Payments` -> `Settings`.
- Scroll down to `Payments accepted on checkout`.
- Verify the JCB payment method is in the list.
- Verify JCB payment method has no checkbox and cannot be activated.
- Hover the icon and verify the tooltip is displayed saying the JCB payment method is coming soon.
- Verify the fees are `3.6% + 0`.

*

-------------------


**Post merge**
